apt-hook/json-hook.cc

```diff
@@ -266,7 +266,12 @@ ESMContext get_esm_context() {
 }
 
 void print_esm_packages(std::string esm_name, std::vector<std::string> package_names, ESMContext &esm_context) {
-    std::cout << "The following security updates require Ubuntu Pro with '";
+
+    if (esm_name == "esm-apps")
+        std::cout << "Get more security updates through Ubuntu Pro with '";
+    else
+        std::cout << "The following security updates require Ubuntu Pro with '";
+
     std::cout << esm_name;
     std::cout << "' enabled:" << std::endl;
 
```

Comment:
I know it's not technically necessary but I suggest we always use brackets.

Also, since we're doing different messages based on this anyway, we can just put the whole message in the if statement.
```suggestion
    if (esm_name == "esm-apps") {
        std::cout << "Get more security updates through Ubuntu Pro with 'esm-apps` enabled:";
    } else {
        std::cout << "The following security updates require Ubuntu Pro with 'esm-infra' enabled";
    }
```

Also, we could change the parameter of this function from `std::string esm_name` to an enum like: `enum ESMType {APPS, INFRA};`

Reply:
@orndorffgrant Good suggestions. I have updated the code